widget.scss

```diff
@@ -0,0 +1,20 @@
+.widgetContainer {
+	display: grid;
+	grid-template-rows: auto 1fr;
+	grid-template-columns: 100px 100px 100px;
```

Comment:
I'm not sure what needs to be displayed in this grid, but the layout approach to rows and columns does seem a little unorthodox - perhaps it would be worth a discussion?